lib/ExpensiMark.js

```diff
@@ -162,6 +162,11 @@ export default class ExpensiMark {
                 },
                 replacement: g1 => `<blockquote>${g1}</blockquote>`,
             },
+            {
+                name: 'heading1',
+                regex: /^#(?!#) +(.*)$/gm,
```

Comment:
```suggestion
                regex: /^#(?!#) +(.+)$/gm,
```

I think this should only match when we have at least one character after the `# `. Otherwise this might get translated into just `<h1></h1>` which is an invalid comment in the backend. [See what I found here for more](https://github.com/Expensify/App/issues/8720#issuecomment-1105771905)

I think we should also add a test for this to ensure parsing "# " just outputs to "# ".

Reply:
yeah good catch! Thanks Andrew!